packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

```diff
@@ -26,30 +26,321 @@ REST endpoints to configure objects whenever you need. For example:
+
+The API key (API token) isn't interchangeable with an Okta [session token](/docs/reference/api/authn/#session-token), access tokens, or ID tokens used with [OAuth 2.0 and OpenID Connect](/docs/reference/api/oidc/).
+
+You can now interact with Okta APIs using scoped OAuth 2.0 access tokens for a number of Okta endpoints. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains. For more information, see [OAuth 2.0 for Okta APIs](/docs/guides/implement-oauth-for-okta/).
+
+### Pagination
+
+Requests that return a list of objects may support pagination. Pagination is based on a cursor and not on page number. The cursor is opaque to the client and specified in either the `before` or `after` query parameter. For some objects, you can also set a custom page size with the `limit` parameter.
+
+Note that for technical reasons not all APIs respect pagination or the `before` and `limit` parameters, see the [Events API](/docs/reference/api/events/) for example.
+
+| Param    | Description                                                                             |
+| -------- | ------------                                                                            |
+| `before` | This is the cursor that points to the start of the page of data that has been returned. |
+| `after`  | This is the cursor that points to the end of the page of data that has been returned.   |
+| `limit`  | This is the number of individual objects that are returned in each page.                |
+
+### Link header
+
+Pagination links are included in the [Link header](http://tools.ietf.org/html/rfc8288) of responses. It is important to follow these Link header values instead of constructing your own URLs as query parameters or cursor formats may change without notice.
+
+``` http
+HTTP/1.1 200 OK
+link: <https://${yourOktaDomain}/api/v1/logs?limit=20>; rel="self"
+link: <https://${yourOktaDomain}/api/v1/logs?limit=20&after=1627500044869_1>; rel="next"
+```
+
+The possible `rel` values are:
+
+| Link relation type | Description                                              |
+| ------------------ | ------------                                             |
+| `self`             | Specifies the URL of the current page of results         |
+| `next`             | Specifies the URL of the immediate next page of results |
+
+When you first make an API call and get a cursor-paged list of objects, the end of the list is the point at which you don't receive another `next` link value with the response. This holds true for all cases, except for the [System Log API](/docs/reference/api/system-log/), where the `next` link always exists in [System Log](/docs/reference/api/system-log/#list-events) polling queries. A polling query is defined as an `ASCENDING` query with an empty or absent `until` parameter, providing a stream of data.
```

Comment:
```suggestion
When you first make an API call and get a cursor-paged list of objects, the end of the list is the point where you don't receive another `next` link value with the response. This holds true for all cases, except for the [System Log API](/docs/reference/api/system-log/) where the `next` link always exists in [System Log](/docs/reference/api/system-log/#list-events) polling queries. A polling query is defined as an `ASCENDING` query with an empty or absent `until` parameter and provides a stream of data.
```

Reply:
I kept the "providing a stream of data" at the end because the polling query IS providing a stream of data.